packages/tasks/src/tasks/object-detection/data.ts

```diff
@@ -70,7 +70,7 @@ const taskData: TaskDataCustom = {
 		},
 		{
 			description: "A cutting-edge object detection application.",
-			id: "Ultralytics/YOLO11",
+			id: "sunsmarterjieleaf/yolov12",
```

Comment:
they are still making yolos lmao

Reply:
it's actually near sota or sota when it comes out